### DISCUSSION TITLE
套件版qb tr的种子目录，mp无法正常读取

### DISCUSSION THREAD

**Discussion by 101letters** [jxxghp/MoviePilot#370] [问题咨询]
### 确认

- [X] 我的版本是最新版本，我的版本号与 [version](https://github.com/jxxghp/MoviePilot/releases/latest) 相同。
- [X] 我已经 [issue](https://github.com/jxxghp/MoviePilot/issues) 中搜索过，确认我的问题没有被提出过。
- [X] 我已经 [Telegram频道](https://t.me/moviepilot_channel) 中搜索过，确认我的问题没有被提出过。
- [X] 我已经修改标题，将标题中的 描述 替换为我遇到的问题。

### 当前程序版本

1.0.8

### 问题类型

插件问题

### 问题描述

设置iyuu插件，按照iyuu同样的映射，在mp中无法正常读取 
![image](https://github.com/jxxghp/MoviePilot/assets/120088769/34a537b4-e3d3-4b76-b4a5-75101ac02f68)

**Comment by dlovew** [comment]
种子目录映射了么？

**Comment by 101letters** [comment]
> 

当然映射 iyuu正常，mp文件夹空白
